tests/manual-test-cases/Group5-Functional-Tests/5-18-Datastore-Cluster-SDRS.robot

```diff
@@ -15,13 +15,14 @@
 *** Settings ***
 Documentation  Test 5-18 - Datastore Cluster SDRS
 Resource  ../../resources/Util.robot
-Test Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+#Suite Setup  Wait Until Keyword Succeeds  10x  10m  SDRS Datastore Setup
```

Comment:
Are you purposefully not running these for now? Just a heads up in case you meant to uncomment.

Reply:
Same question/reminder with the `ROBO SKU` suite.

Reply:
Nevermind, I see the comment on line 74 of `5-19-ROBO-SKU.robot`